whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/lireditable/BaseLirEditableAttributes.groovy

```diff
@@ -2,8 +2,9 @@ package net.ripe.db.whois.spec.update.lireditable
 
 
 import net.ripe.db.whois.spec.BaseQueryUpdateSpec
+import org.junit.jupiter.api.Tag
 
-@org.junit.jupiter.api.Tag("IntegrationTest")
```

Comment:
No longer necessary to use fully qualified package, as the `Tag` class doesn't exist any more (there was a tag feature in Whois but we removed it)